dev/src/watch.ts

```diff
@@ -426,6 +433,15 @@ abstract class Watch {
      * Initializes a new stream to the backend with backoff.
      */
     const initStream = () => {
+      if (this._backoff.getRetryCount() > DEFAULT_MAX_RETRY_ATTEMPTS) {
+        closeStream(
+          new Error(
+            'Exceeded maximum number of retries before any ' +
+              'response was received.'
```

Comment:
This could be one line. We don't really have a fixed line length and 80 character wrap is so 1994.